openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -259,22 +263,33 @@ def load_from_module_store(cls, course_id):
                     # other one will cause an IntegrityError because it tries
                     # to save a duplicate.
                     # (see: https://openedx.atlassian.net/browse/TNL-2854).
+                    logger.info(
```

Comment:
Shouldn't this be at least a warning?

Reply:
It's expected to happen fairly often every time `VERSION` is bumped, so IMO it shouldn't be seen as a yellow flag of any sort.